### ISSUE TITLE
Performance personal api

### ISSUE THREAD

**Issue by odin568** [djbrown/spielerplus-calendar#8]
Hi @djbrown ,
I use with success the teams endpoint of your app, which queries quite fast (as it uses the calendar overview html).
The personal API also works but needs (at least on my RaspberryPi) almost 3 minutes to give the response. This is too much for at least my App "ICSx⁵". Did not try how it would behave if I import the calendar via i.e. Google and let google update it.
Do you have any experience for this?
Is it due to my Raspberry infra or too many events? Perhaps parallizing requests might give a bump but of course we should not be detected as a DDoS attack 😄 
Many thanks in advance

**Comment by djbrown** [comment]
Another option could be to store the necessary data on the API server and separate the update process from the serving process.
That would make the operations more  complex, but also increase flexibility.

For parallelizing requests we would have to measure the actual performance impact.

**Comment by odin568** [comment]
Yes, updating in background every i.e. hour and after start and from API always returning the cached instance was also my idea.
